raja-perf/ats4/configs/auxiliary_software_files/compilers.yaml

```diff
@@ -0,0 +1,30 @@
+compilers:
+- compiler:
+    spec: clang@16.0.0-rocm5.5.1
+    paths:
+      cc: /opt/cray/pe/craype/2.7.21/bin/cc
+      cxx: /opt/cray/pe/craype/2.7.21/bin/CC
+      f77: /opt/cray/pe/craype/2.7.21/bin/ftn
+      fc: /opt/cray/pe/craype/2.7.21/bin/ftn
+    flags: # TODO: Fix spack concretization bug
+      cflags: -g -O2
+      cxxflags: -g -O2 -std=c++14
+      fflags: -g -O2
+    operating_system: rhel8
+    target: x86_64
+    modules: [StdEnv, PrgEnv-cray/8.4.0, rocm/5.5.1, craype/2.7.21, craype-accel-amd-gfx90a, cce/16.0.0] # TODO: Make available to ramble or remove entirely
```

Comment:
What does it mean to make it available to ramble?

Reply:
We do not need this file anymore. This has been replaced by $benchpark/configs/ats4/auxiliary_software_files/compilers.yaml